src/analysis/gMCS/GPR2models.m

```diff
@@ -75,6 +81,7 @@
 elseif numWorkers == 1
     poolsize = 0;               % Single core
 else
+    gcp();
```

Comment:
Wouldn't this be `parpool(numWorkers)`? Also, it would be `numWorkers`, not `NumWorkers`.